### DISCUSSION TITLE
in_progress event is not being delivered

### DISCUSSION THREAD

**Discussion by sandokandias** [community/community#53334] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

Since this morning the event `in_progress` is not being delivered for any workflow run.
Only the events `requested` and `completed` are being delivered.

![Screen Shot 2023-04-20 at 15 42 19](https://user-images.githubusercontent.com/610003/233458509-3197ce7b-7756-46a8-8f04-12ff97159517.png)

**Comment by sandokandias** [comment]
Update: I picked an old (2023-03-29) in_progress event and click in redelivery, the event was delivered successful.


![Screen Shot 2023-04-20 at 16 05 04](https://user-images.githubusercontent.com/610003/233463464-52e9395f-b1f1-4585-b715-7b0cd497b19c.png)

**Comment by smithw-adsk** [comment]
I'm also seeing a similar thing. We have actions running and in progress, but the query fails

https://api.github.com/repos/Autodesk/maya-usd/actions/workflows/maya-usd-preflight-launcher.yml/runs
https://api.github.com/repos/Autodesk/maya-usd/actions/workflows/maya-usd-preflight-launcher.yml/runs?status=in_progress

**Reply by smithw-adsk** [reply]
```
{
  "total_count": 2581,
  "workflow_runs": [
    {
      "id": 4758157717,
      "name": "Pre-flight build on pull request",
      "node_id": "WFR_kwLOC9rQmM8AAAABG5u5lQ",
      "head_branch": "perrauo/HYDRA-324/revise-proxy-shape-scene-index-to-match-new-scene-index-api-2",
      "head_sha": "fa0626538cdab1051001f0bfccc477d3bec36c99",
      "path": ".github/workflows/maya-usd-preflight-launcher.yml",
      "display_title": "HYDRA-324: update proxy shape scene index to support interpret pick result",
      "run_number": 3536,
      "event": "pull_request",
      "status": "in_progress",
      "conclusion": null,
      "workflow_id": 1438190,
      "check_suite_id": 12378897542,
      "check_suite_node_id": "CS_kwDOC9rQmM8AAAAC4db8hg",
      "url": "https://api.github.com/repos/Autodesk/maya-usd/actions/runs/4758157717",
      "html_url": "https://github.com/Autodesk/maya-usd/actions/runs/4758157717",
```

```
{
  "total_count": 0,
  "workflow_runs": [

  ]
}
```

**Comment by sandokandias** [comment]
Update: The `in_progress` event is still not being delivered by Webhooks since 2023-04-20.


![Screen Shot 2023-04-21 at 15 19 05](https://user-images.githubusercontent.com/610003/233708756-6da5b2fd-67b5-4a18-ae1d-24216d539e59.png)

**Comment by EmilRinggaard-zoetis** [comment]
We are experiencing the same issue. We have a system that relies in the workflow.in_progress event, so we might have to patch our backend for some temporary stuff before github fixes the issue.
